modules/util/source-formatter/src/main/resources/sourcechecks.xml

```diff
@@ -1625,6 +1625,11 @@
 			<description name="Reorder parameters in the getItems and getItemsCount methods of the FDSDataProvider interface" />
 			<weight>1</weight>
 		</check>
+		<check name="UpgradeJavaGetLayoutDisplayPageObjectProviderCheck">
+			<category name="Upgrade" />
+			<description name="Replace the parameter type of the getLayoutDisplayPageObjectProvider method" />
```

Comment:
@michaelmdelcavalcanti, can you please specify which parameter will be replaced? I mean, can you please give a little more details about it?